src/hw3/HW3.java

```diff
@@ -0,0 +1,46 @@
+package hw3;
+
+public class HW3 {
+
+    static String max(String par) {
```

Comment:
According to the home task 3 String parameters should be
(String word1, String word2, String word3 )